### PR TITLE
Newsletter 9: Add a bunch of blog posts

### PR DESCRIPTION
**Pull request by ozkriff**
@bitshifter  @hoj-senna @snorrwe @MasonRemaley r?

_Part of #111 ("Newsletter 9: Coordination/Tracking")_

### CODE REVIEW

File: content/posts/newsletter-009/index.md [hunk 1]
```diff
@@ -388,6 +388,79 @@ send her a [DM][oliviff-dm].
...
+[ash-tutorial]: https://hoj-senna.github.io/ashen-aetna
+[ash]: https://github.com/MaikKlein/ash
+
+## [Implementing a Linear Quadtree in Rust][quadtree-post]
+
+![demo of how the Z-order curve fills the space](quadtree.jpeg)
+
+[@snorrwe] has written [a blog post][quadtree-post]
+about exploring an implementation of a linear quadtree
+and comparing it with a naive implementation of spacial data quering.
```
Comment (snorrwe) [inline content/posts/newsletter-009/index.md:429]: There's a typo, should by querying I think (?).

**Comment by snorrwe** [comment]
Thank you very much :)

File: content/posts/newsletter-009/index.md (hunk 1 above)
Comment (ozkriff) [reply content/posts/newsletter-009/index.md]: thanks, fixed

**Comment by MasonRemaley** [comment]
Thanks for thinking to include me! :)

**Pull request merged**